srcs/request_handling/FixedResponses.cpp

```
@@ -94,3 +94,12 @@ std::unique_ptr<ResponsePacket> created(const std::string &location)
 	response_packet->set_final_response();
 	return response_packet;
 }
+
+std::unique_ptr<ResponsePacket> not_implemented()
```

cppcheck

[style] The function 'not_implemented' is never used.

(unusedFunction)